dipy/viz/horizon/tab/roi.py

```diff
@@ -1,52 +1,106 @@
-from dipy.utils.optpkg import optional_package
-from dipy.viz.horizon.tab import (HorizonTab, build_label, color_single_slider)
-
-fury, has_fury, setup_module = optional_package('fury', min_version="0.10.0")
-
-if has_fury:
-    from fury import ui
+from dipy.viz.horizon.tab import (HorizonTab, build_slider, build_checkbox)
 
 
 class ROIsTab(HorizonTab):
     def __init__(self, contour_actors):
-        self.__actors = contour_actors
-        self.__name = 'ROIs'
+        """Initialize interaction tab for ROIs visualization.
+
+        Parameters
+        ----------
+        contour_actors : list
```

Comment:
list of ? string ? actors? 
Can you add a description below also

Reply:
Updated!